### PR TITLE
do not remove the old framework, because that's now the renderer

### PR DESCRIPTION
**Pull request by ndelangen**
I saw an auto-migration remove `@stroybook/react` that is incorrect.

Although this package is no longer the 'framework' it is the renderer, and users can import APIs from it, so it should remain a dependency for users.

### CODE REVIEW

**Comment by yannbf** [comment]
Even though that's correct, shouldn't users change their imports from @storybook/react to @storybook/react-webpack5?

**Comment by ndelangen** [comment]
> Even though that's correct, shouldn't users change their imports from @storybook/react to @storybook/react-webpack5? If so, we could print out on the banner that users should change their imports too

No, they should not.

Users should keep importing from `@storybook/react`.

Importing from `@storybook/react-webpack5` is only for config.

**Review by yannbf — CHANGES_REQUESTED** [review]
Got it! Can you please update the tests?

File: code/lib/builder-webpack5/src/preview/iframe-webpack.config.ts [hunk 1]
```diff
@@ -2,6 +2,7 @@ import path from 'path';
 import { DefinePlugin, HotModuleReplacementPlugin, ProgressPlugin, ProvidePlugin } from 'webpack';
 import type { Configuration } from 'webpack';
 import HtmlWebpackPlugin from 'html-webpack-plugin';
+// @ts-ignore
```
Comment (ndelangen) [inline code/lib/builder-webpack5/src/preview/iframe-webpack.config.ts:5]: This started throwing an error at build, no idea why.

I tried installing the `@types/..` package but it's not using the right version of webpack, so it's un-usable..

**Pull request merged**